src/refbox/main.cpp

```diff
@@ -42,29 +42,17 @@
 #	include <mongocxx/instance.hpp>
 #endif
 
-#include <termios.h>
-
 using namespace llsfrb;
 
 int
 main(int argc, char **argv)
 {
-	// Disable annoying display of ^C on Ctrl-C
```

Comment:
What was the purpose of this originally and why was it removed?

Reply:
I suppose exactly what the text states (not displaying ^C if you kill the refbox).
The reason why it is removed is, that if the refbox shuts down without restoring the termios, then this affects your running terminal that executes the refbox. In fedora34 this lead to you not being able to see what you type anymore. In Fedora 33 i did not encounter that issue...